src/pages/SingleTweet.tsx

```diff
@@ -0,0 +1,224 @@
+import { Dropdown } from "react-bootstrap";
```

Comment:
remove this file, and use SingleTweet that available from develop

Reply:
done